src/lib/support/crypto/EllipticCurve.h

```diff
@@ -91,6 +98,11 @@ class EncodedECDSASignature
 class EncodedECPrivateKey
 {
 public:
+    enum
+    {
+        kMaxValueLength = (Platform::RoundUp(WEAVE_CONFIG_MAX_EC_BITS, 8) / 8) + 1
```

Comment:
It looks like some of the vestigal, non-macro math got left behind here (that is, "/ 8) + 1").

Reply:
See prior comment.